src/screen.js

```diff
@@ -2,51 +2,51 @@
 
 /**
  *  Screen handles calculator screen.  
- *  Constructor.
- *  @param {String} element.
  */
-function Screen(element) {
-    this.element = element;
-    this.reset = false;
-}
-
-/**
- * Allows a new entry. 
- */
-Screen.prototype.resetOnNextInput = function() {
-    this.reset = true;
-}
-
-Screen.prototype.resetOfNextInput = function() {
-    this.reset = false;
-}
+class Screen {
+
+    /**
+     *  Constructor.  
+     *  @param {String} element.
+     */
+    constructor(element) {
+        this.element = element;
+        this.reset = false;
+    }
 
-/**
- * Sets the given number on the screen 
- * @param {Number} number
- */
-Screen.prototype.setNumber = function(number) {
-    this.element.value = number;
-}
+    /**
+     * Allows a new entry. 
+     */
+    resetOnNextInput() {
+        this.reset = true;
+    }
 
-/**
- * Gets the current number from the screen. 
- */
-Screen.prototype.getNumber = function() {
-    return this.element.value;
-};
+    /**
+     * Sets the given number on the screen 
+     * @param {Number} number
+     */
+    setNumber(number) {
```

Comment:
ES6 introduces getters and setters, which are designed to purposes like this one. From the outside it looks like you are working with a number:
```
screen.number = 5;
```
While in reality you are using a function to do that:
```
set number(value) {
 this.element.value = value;
}
```
Please investigate how getters / setters are used / defined, and what is their purpose. Once you understand, update this class with your findings.